controller/common/output.go

```diff
@@ -244,6 +244,63 @@ func NewWebHook(url, target string) *Webhook {
 	return w
 }
 
+func getDetailInfoFromLog(elog interface{}) string {
+	details := ""
+	v := reflect.ValueOf(elog).Elem()
+
+	if field := v.FieldByName("ClusterName"); field.IsValid() && field.Kind() == reflect.String {
+		details += fmt.Sprintf("cluster: %s, ", field.String())
+	}
```

Comment:
To parse the `elog`, maybe this library can help? https://github.com/mitchellh/pointerstructure